specification/nhais-adaptor.yaml

```diff
@@ -78,104 +85,41 @@ paths:
           application/json:
             schema:
               $ref: components/schemas/Parameters.yaml
-            examples:
-              deduction:
-                summary: Deduction - Death
-                value:
-                  $ref: components/examples/requests/NHAIS/deduction-death.json
       responses:
         '202':
-          description: Deduction forwarded to NHAIS for processing. Respose body is empty.
```

Comment:
spelling - Respose

Reply:
thanks for the spellcheck, doesn't seem to work in the yaml file